bench/state_transition/process_block.zig

```diff
@@ -0,0 +1,351 @@
+const std = @import("std");
+const zbench = @import("zbench");
+const state_transition = @import("state_transition");
+const types = @import("consensus_types");
+const config = @import("config");
+const preset = state_transition.preset;
+
+// printf "Date: %s\nKernel: %s\nCPU: %s\nCPUs: %s\nMemory: %sGi\n" "$(date)" "$(uname -sr)" "$(sysctl -n machdep.cpu.brand_string)" "$(sysctl -n hw.ncpu)" "$(echo "$(sysctl -n hw.memsize) / 1024 / 1024 / 1024" | bc)"
+// Date: Tue Dec  9 2025
+// Kernel: Darwin 25.1.0
+// CPU: Apple M3
+// CPUs: 8
+// Memory: 16Gi
+//
+// zbuild run bench_process_block -Doptimize=ReleaseFast
+//
+// benchmark              runs     total time     time/run (avg ± σ)     (min ... max)                p75        p99        p995
+// -----------------------------------------------------------------------------------------------------------------------------
+// process_block_header   61       2.653s         43.494ms ± 40.773ms    (26.787ms ... 306.582ms)     37.671ms   306.582ms  306.582ms
+// process_withdrawals    54       1.751s         32.434ms ± 10.277ms    (26.843ms ... 99.635ms)      32.642ms   99.635ms   99.635ms
+// process_execution_payl 48       1.556s         32.432ms ± 5.087ms     (26.721ms ... 44.789ms)      36.231ms   44.789ms   44.789ms
+// process_randao         64       2.076s         32.443ms ± 23.425ms    (27.134ms ... 212.965ms)     29.506ms   212.965ms  212.965ms
+// process_eth1_data      71       2.001s         28.192ms ± 1.228ms     (26.04ms ... 32.299ms)       28.856ms   32.299ms   32.299ms
+// process_operations     44       2.401s         54.581ms ± 33.324ms    (41.66ms ... 254.774ms)      51.213ms   254.774ms  254.774ms
+// process_sync_aggregate 67       2.086s         31.145ms ± 10.657ms    (26.877ms ... 111.953ms)     30.532ms   111.953ms  111.953ms
+
+const CachedBeaconStateAllForks = state_transition.CachedBeaconStateAllForks;
+const BeaconStateAllForks = state_transition.BeaconStateAllForks;
+const SignedBlock = state_transition.SignedBlock;
+const SignedBeaconBlock = state_transition.SignedBeaconBlock;
+const Body = state_transition.Body;
+const st_mod = @import("state_transition").state_transition;
+const stateTransition = st_mod.stateTransition;
```

Comment:
we already imported `state_transition` above, so

```suggestion
const stateTransition = state_transition.state_transition.stateTransition;
```

although we should probably do something about this ugly namespacing (in a separate PR)